webpack.config.js

```diff
@@ -9,6 +9,10 @@ var paths = require('./scripts/paths.js');
 var englishStrings = require('./locales/en-US.json');
 var currencies = require('./data/currencies.js');
 var url = require('url');
+var Habitat = require('habitat');
```

Comment:
Yeah he walked me through this. Right now we oonly use habitat inside server.js.

During a npm install, we run webpack without server.js, thus the envs are undefined.